node/cmd/guardiand/node.go

```diff
@@ -947,6 +943,10 @@ func runNode(cmd *cobra.Command, args []string) {
 	// Redirect ipfs logs to plain zap
 	ipfslog.SetPrimaryCore(logger.Core())
 
+	// Database
+	db := db.OpenDb(logger.With(zap.String("component", "badgerDb")), dataDir)
```

Comment:
There are other log messages from the lines above that will not be sent to loki.  Do you want to move up the loki logging startup instead of moving the db start down?

Reply:
I made this change but it required moving some other stuff. Please inspect these changes very carefully to make sure I didn't mess anything up.